src/services/wopi/index.js

```diff
@@ -16,6 +16,10 @@ const filePostActionHelper = require('./utils/filePostActionHelper');
 const handleResponseHeaders = require('../../middleware/handleResponseHeaders');
 
 const wopiPrefix = '/wopi/files/';
+const fileByIdQuery = (fileId) => {
```

Comment:
would be great to find a way to overwrite findOne, at the model level
There are more use cases for FileModel.findOne